src/App.jsx

```diff
@@ -0,0 +1,22 @@
+import { Route, Switch } from "wouter";
+
+import Home from './components/Home'
+import Login from './pages/Login'
+import Registration from './pages/Registration'
+import { UserContextProvider } from './contexts/UserContext'
+
+function App() {
+  return (
+    <>
+      <UserContextProvider>
+        <Switch>
+          <Route component={Home} path="/" />
+          <Route component={Login} path="/login" />
+          <Route component={Registration} path="/registration" />
```

Comment:
You can create an array of objects with component & path, and the map through it displaying only one `Route`, that can make your code a little bit more maintainable. Also, its a good practice to use constants for the paths, so you avoid typing mistakes

```suggestion
          {routes.map(({component,path})=> <Route component={component} path={path}/>}
```